x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+
+import './result.scss';
+
+import { EuiPanel, EuiIcon } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { FieldValue, Result as ResultType } from './types';
+import { ResultField } from './result_field';
+import { ResultHeader } from './result_header';
+
+interface Props {
+  result: ResultType;
+  showScore?: boolean;
+}
+
+const RESULT_CUTOFF = 5;
+
+export const Result: React.FC<Props> = ({ result, showScore }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const ID = 'id';
+  const META = '_meta';
+  const resultMeta = result[META];
+  const resultFields = Object.entries(result).filter(([key]) => key !== META && key !== ID);
+  const numResults = resultFields.length;
+
+  return (
+    <EuiPanel
+      paddingSize="none"
+      key="results"
+      className={'appSearchResult'}
```

Comment:
Minor nit, this can just be a static string
```suggestion
      className="appSearchResult"
```